### DISCUSSION TITLE
Address book plugin

### DISCUSSION THREAD

**Discussion by vdibart** [TiddlyWiki/TiddlyWiki5#5316] [Q&A]
Hello,

I'm so happy to see TiddlyWiki still alive and kicking.

A lifetime ago I built an address plugin that was pretty full featured, customizable, hackable, searchable, etc.  Supported import and export in CSV format. 

I have no idea how out of date it or its dependencies are but if anyone here is interested to take a look and possibly maintain it I'm happy to post the source code.  I used it exclusively for a long time before finally moving everything into my phone.

Let me know if there's a better place to direct this.  Thanks.

-vdibart

**Comment by pmario** [comment]
Hi. Is it TWclassic or TW 5?

**Reply by kookma** [reply]
If it is a TW5 plugin, then I am sure it worth to publish it.
